src/setup.cfg

```diff
@@ -6,13 +6,11 @@ max-complexity = 11
 
 [isort]
 combine_as_imports = true
-default_section = THIRDPARTY
 include_trailing_comma = true
 known_first_party = pretix
 known_third_party = versions
-known_standard_library = typing,enum,mimetypes
+extra_standard_library = typing,enum,mimetypes
 multi_line_output = 5
-not_skip = __init__.py
```

Comment:
I took a look at https://timothycrosley.github.io/isort/docs/upgrade_guides/5.0.0/ for this. With and without these changes my isort complains about the order in 20 or so py files :/